### DISCUSSION TITLE
API Request to release nodeSecret for Node Locked model

### DISCUSSION THREAD

**Discussion by MinHtet-O** [Labs64/NetLicensing-Community#86] [Questions] [answered]
Hello,

I'm new to NetLicensing here. I have defined Node Locked licensing model and registered two nodes with nodeSecret key. Is there anyway I can release my nodeSecret automatically from API?

Best,
Min
![netlicensing](https://user-images.githubusercontent.com/33474727/122888269-b4012c00-d367-11eb-9e4c-8c8dbe29ca26.PNG)

**Comment by r-brown** [comment, marked as answer]
Hello @MinHtet-O - yes this possible and described in the #30
